### PR TITLE
Add Ory stack configs

### PR DESCRIPTION
**Pull request by mpgxvii**
Description:

- Updates existing `kratos` config
- Adds new Ory components: `radar-self-enrolment-ui` and `hydra`
- Also has the Kratos -> MP webhook configs
- Docker image for self-enrolment UI is here temporarily: `mpgxvii/radar-self-enrolment-ui:latest`

#### Checklist:
- [ ] The [Main workflow](https://github.com/RADAR-base/ManagementPortal/actions/workflows/main.yml) has succeeded
- [ ] The [Gatling tests](https://github.com/RADAR-base/ManagementPortal#other-tests) have passed
- [ ] I have logged into the portal running locally with default admin credentials
- [ ] I have updated the README files if this change requires documentation update
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Bdegraaf1234** [comment]
I am seeing other PRs merge into this branch, which will be merged into dev here, is that what we want? If we merge this after merging the other PRs we will break the current flow, as they require hydra.

I would say we merge into one of the two feature branches for hydra/ory

**Review by yatharthranjan — APPROVED** [review]
Looks good. 

I have updated to use a single instance of Postgres for both Kratos and Hydra in an effort to reduce the resource load for local dev and testing.

BTW, I am getting a csrf security violation when signing up (http 403). I assume that is because the enrolment ui docker image is not up to date with you latest changes?

**Comment by mpgxvii** [comment]
> I am seeing other PRs merge into this branch, which will be merged into dev here, is that what we want? If we merge this after merging the other PRs we will break the current flow, as they require hydra.
> 
> I would say we merge into one of the two feature branches for hydra/ory

@Bdegraaf1234 @yatharthranjan Yes sure, I've updated it now to merge to `feature/ory-based-authorization`.

**Pull request merged**
